src/libsystemd-network/lldp-internal.h

```diff
@@ -12,6 +14,7 @@ struct sd_lldp {
         unsigned n_ref;
 
         int ifindex;
+        char ifname[IFNAMSIZ + 1];
```

Comment:
So this assumes we will not need to use alternative names here. Shouldn't we also allow that?